main_test.go

```diff
@@ -118,6 +120,40 @@ func TestPackageIndex(t *testing.T) {
 	}
 }
 
+// TestAllPackageIndex generates and compares all index.json files for the test packages
+func TestAllPackageIndex(t *testing.T) {
+	publicPath := "./testdata/public"
```

Comment:
Hm, does it still work like that? There should be no `public` directory anymore: https://github.com/ruflin/package-registry/tree/json-files/testdata . As there is no need for building, you should be good to go with sth like this:

```golang
packagesBasePaths := []string{"./testdata/package"}
```

Reply:
ups, I'm glad you spotted this. It worked because the dir was just empty so no files were generated and the old files were already there :-( Fixed now.